### PR TITLE
Update PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by adizere**
Updated the PR template to put a reminder, so that when we do new version releases we notify the comms team about it in advance.

______

### PR author checklist:
- [X] Tagged reviewers

### Reviewer checklist:

- [ ] Reviewed `Files changed` in the GitHub PR explorer.
- [ ] Manually tested (in case integration/unit/mock tests are absent).

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (rnbguy) [inline .github/PULL_REQUEST_TEMPLATE.md]: It appears to be a link to a restricted page. Should it stay public?

Comment (adizere) [reply .github/PULL_REQUEST_TEMPLATE.md]: it should be a private page, shared with Andrea, then he will manage making the public communications out of that content in that private page.

Comment (rnbguy) [reply .github/PULL_REQUEST_TEMPLATE.md]: I meant, should the link address be visible to the public - even though not everyone can open it and read its content ? 😄

**Review by Farhad-Shabani — COMMENTED** [review]
Appreciate adding this @adizere :pray:
I've the same comment as Rano's. I'd suggest not pointing to a private page since it'll be in all PRs. What about adding it as an initial step into the [RELEASES.md](https://github.com/cosmos/ibc-rs/blob/main/RELEASES.md)?

**Comment by adizere** [comment]
Does this fix it @rnbguy @Farhad-Shabani ? [d8ca5c6](https://github.com/cosmos/ibc-rs/pull/1067/commits/d8ca5c6f6757ec2a1977e7377a9b4b4733e770b2)

File: RELEASES.md
Comment (rnbguy) [inline RELEASES.md]:
```suggestion
### Communications (non-technical) release pipeline
```

File: RELEASES.md
Comment (rnbguy) [inline RELEASES.md]:
```suggestion
- Notify the communications team about the pending release and prepare an announcement.
- Coordinate with other organizations that are active in IBC development (e.g., Interchain) and keep them in the loop.

```

**Review by rnbguy — COMMENTED** [review]
@adizere thanks very much for the PR :pray:

I suggested some changes. I will wait for Farhad to give the final approval.

**Review by Farhad-Shabani — APPROVED** [review]
🙌

**Pull request merged**
